package.json

```diff
@@ -13,6 +13,9 @@
   ],
   "author": "Misko Hevery <misko@hevery.com>",
   "license": "MIT",
+  "engines": {
+    "node": ">= 8.0.0"
```

Comment:
```suggestion
    "node": ">= 10"
```

Reply:
and use [`npm:please-upgrade-node`](https://www.npmjs.com/package/please-upgrade-node) to enforce minimum Node.js requirement more strongly (#444)